src/System.Reflection.Metadata/Directory.Build.props

```diff
@@ -1,7 +1,6 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>1.9.0</PackageVersion>
     <AssemblyVersion>1.4.4.0</AssemblyVersion>
```

Comment:
@ericstj @Anipik what was the plan regarding assembly versions? I think we want to keep them in sync with the package version going forward?

Reply:
The assembly version needs to be updated for all packages. I am working on that change.
We dumped up the package version for all the rest of the packages.